app/controllers/issues/evidence_controller.rb

```diff
@@ -114,6 +120,10 @@ def set_auto_save_key
     end
   end
 
+  def set_evidence
+    @evidence = Evidence.find_by!(id: params[:id], issue_id: params[:issue_id])
```

Comment:
```suggestion
    @evidence = Evidence.find_by(id: params[:id], issue_id: params[:issue_id])
```

We don't want to raise an exception if this isn't found, instead it should be handled gracefully. In this case it probably makes sense to have `@evidence` be `nil` if it's not valid since we're only using it for `liquid_resource_assigns`

Reply:
Everywhere else in this controller/this controller's views we're getting to the evidence through the issue or the affected_nodes. We have an `:issue_id` param so should we get the evidence via the issue instead? That will ensure that it exists